pom.xml

```diff
@@ -5,10 +5,11 @@
     <artifactId>scmdb</artifactId>
     <packaging>jar</packaging>
     <name>scmdb</name>
-    <version>1.8-SNAPSHOT</version>
+    <version>1-SNAPSHOT</version>
```

Comment:
why 1-SNAPSHOT? Did version naming schema was changed for scmdb?

Reply:
yes, I've moved it to the MASTER version, so the actual version number doesn't matter anymore and I'm not going to update it. 1-SNAPSHOT is short version of 1.0-SNAPSHOT